### ISSUE TITLE
Mentions no longer showing up in Comment area.

### ISSUE THREAD

**Issue by MacsOpinion** [pfefferle/wordpress-webmention#502]
I have two mentions showing in the Admin area for this post https://macsopinion.com/trudeaus-christmas-tax-break-miracle/
They are not showing in the actual post comments area.
There are several mentions in various posts that also do not display.

Current WP version: 6.7.1
ChromeNews ProVersion: 5.3.0
Self Hosted

**UPDATE:** I unchecked the " Separate Webmention Types from Comments" and they will show up within the comments. But I don't like that. Before they would be listed as Mentions above the comment section. Can we get that back?

**Comment by pfefferle** [comment]
This was never gone, but with the new FSE themes, it got more and more complicated to find the right hook! Can you tell us what theme you use?

**Comment by MacsOpinion** [comment]
ChromeNews ProVersion: 5.3.0

**Comment by ndegruchy** [comment]
Yeah, I'm not seeing this form on twentytwentyfive. I'm not sure what needs to be done, as there are two comment components in the FSE. Askimet reliably appends the privacy notice to the form regardless of how it's used. Maybe look at that?
